web/themes/joinup/prototype/sass/components/_form.sass

```diff
@@ -98,5 +99,33 @@
 .form__description--checkbox
   margin-left: 25px
 
+// Dependencies
+// --------------------------------------------------
+.form__actions
```

Comment:
What are these changes related to? If they are the ones from the drupal hacks, why moving them here as they are still very specific to the drupal markup?